sonar/config_sonar.py

```diff
@@ -591,3 +591,16 @@
         'default': 'https://nbn-resolving.org/_identifier_'
     }
 }
+
+# URN
+# ===
+# Config to generate URN identifiers for organisations
+SONAR_APP_DOCUMENT_URN = {
+    'organisations': {
```

Comment:
I would comment the configuration with some description. The "real" configuration should be set during the deployment.